content/source/docs/cloud/api/run.html.md

```diff
@@ -79,20 +92,21 @@ This POST endpoint requires a JSON object with the following properties as a req
+---------------------------------------------------|---------------------|----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`data.attributes.auto-apply`                       | bool                | (The current Auto Apply option of the workspace)   | Whether to automatically apply changes when a Terraform plan is successful. Defaults to the [workspace Auto Apply](https://www.terraform.io/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply) setting.
+`data.attributes.is-destroy`                       | bool                | false                                              | Specifies if this plan is a destroy plan, which will destroy all provisioned resources. Mutually exclusive with `refresh-only`.
+`data.attributes.message`                          | string              | "Queued manually via the Terraform Enterprise API" | Specifies the message to be associated with this run.
+`data.attributes.refresh`                          | bool                | true                                               | Specifies whether or not to refresh the state before a plan.
+`data.attributes.refresh-only`                     | bool                | false                                              | Whether this run should use the refresh-only plan mode, which will refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.
+`data.attributes.replace-addrs`                    | array[string]       | (nothing)                                          | Specifies an optional list of resource addresses to be passed to the `-replace` flag.
+`data.attributes.target-addrs`                     | array[string]       | (nothing)                                          | Specifies an optional list of resource addresses to be passed to the `-target` flag.
+`data.attributes.variables`                        | array[{key, value}] | empty                                              | Specifies an optional list of run-specific variable values. See [Run-Specific Variables](https://www.terraform.io/docs/cloud/workspaces/managing-variables.html#run-specific-variables) in the Terraform Cloud workspaces documentation for more information.
+`data.relationships.workspace.data.id`             | string              | (nothing)                                          | Specifies the workspace ID where the run will be executed.
+`data.relationships.configuration-version.data.id` | string              | (nothing)                                          | Specifies the configuration version to use for this run. If the `configuration-version` object is omitted, the run will be created using the workspace's latest configuration version.
```

Comment:
Is (nothing) a convention we usually use? If so I accept it. If not, I wonder is there a convention we typically use for no default. Or, should we just leave that cell blank?